### ISSUE TITLE
 zendframework/zend-stratigility ^1.1 dependancies 

### ISSUE THREAD

**Issue by h3christophe** [psr7-sessions/storageless#68]
I am using  zendframework/zend-stratigility version ^2.0 (2.0.1)
And because of that I cannot install this package 

```
  - psr7-sessions/storageless 3.0.0 requires zendframework/zend-stratigility ^1.1 -> satisfiable by zendframework/zend-stratigility[1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.2.0, 1.2.1, 1.3.0, 1.3.1, 1.3.2, 1.3.3].
    - Can only install one of: zendframework/zend-stratigility[1.1.0, 2.0.1].
    - Can only install one of: zendframework/zend-stratigility[1.1.1, 2.0.1].
    - Can only install one of: zendframework/zend-stratigility[1.1.2, 2.0.1].
    - Can only install one of: zendframework/zend-stratigility[1.1.3, 2.0.1].
    - Can only install one of: zendframework/zend-stratigility[1.2.0, 2.0.1].
    - Can only install one of: zendframework/zend-stratigility[1.2.1, 2.0.1].
    - Installation request for zendframework/zend-stratigility (locked at 2.0.1) -> satisfiable by zendframework/zend-stratigility[2.0.1].
```

Is there a workaround ? 

Thank you

**Comment by Ocramius** [comment]
See https://github.com/psr7-sessions/storageless/pull/59

If you want, you can send a patch to allow stratigility 2.x in `composer.json`, and then we'll look at whether that works.

**Comment by Ocramius** [comment]
This was already solved in 3.0.0
